hack/prepare-minor-release.sh

```diff
@@ -36,7 +36,7 @@ fi
 ic_version=$1
 helm_chart_version=$2
 
-prev_ic_version=$(echo $ic_version | awk -F. '{ printf("%s.%s.%d", $1, $2, $3-1) }')
+prev_ic_version=$(echo $ic_version | awk -F. '{ printf("%s\\.%s\\.%d", $1, $2, $3-1) }')
 prev_helm_chart_version=$(echo $helm_chart_version | awk -F. '{ printf("%s.%s.%d", $1, $2, $3-1) }')
```

Comment:
is it possible that the same problem can appear for helm chart version replacement as well?